src/openassetio-python/bridge/CMakeLists.txt

```diff
@@ -68,13 +71,11 @@ target_include_directories(openassetio-python-bridge
 
 # Library dependencies.
 target_link_libraries(openassetio-python-bridge
-    PRIVATE
+    PUBLIC
     openassetio-core
-    # TODO(DF): Pybind docs say the pybind11 target just includes
-    #   pybind and Python headers, which is all we should need for this
-    #   library. However, a bug in the ConanCenter package means all
-    #   pybind targets include everything. See #528.
-    pybind11::pybind11)
+    PRIVATE
+    pybind11::module
+    pybind11::windows_extras)
```

Comment:
I had to look up this target added.

It adds /bigobj, which is a little surprising.

Why was it needed?

Reply:
:grin: no idea, it was just a blind copy-paste.  I can remove it if its just confusing things.

Reply:
Removed `windows_extras` in 4d338952